### ISSUE TITLE
gtk-canvas installation

### ISSUE THREAD

**Issue by jpellegrini** [egallesio/STklos#57]
I was trying to install the `gtk-canvas` package, but `stklos-pkg` said I don't have goocanvas installed. However, I do...

What happens is that the `configure.in` in `stklos-gtk-canvas` searches for `goocanvas`, and in Debian `pkg-config` has it with the name `goocanvas-2.0` (ouch!)

Is it possible to make the "`goocanvas`" text in `configure.in` configurable? Something like a variable `GOOCANVAS_PKG`, which is "`goocanvas`" by default, but can be overriden?

```
if pkg-config --exists $(GOOCANVAS_PKG) 2> /dev/null
```

Also - is the source of those packages (which are downloadable with `stklos-pkg`) available in a git repository somewhere?
Of course I can open the tarball that is downloaded, but it would be easier if I could track them from git (and send pull requests! :) :)

**Comment by jpellegrini** [comment]
It seems that the situation is a bit trickier than what I thought:


|                | goocanvas | goocanvas-2.0 |
|----------------|-----------|---------------|
|FreeBSD 12.2    | Y         | Y             |
|FreeBSD 13.0    | Y         | N             |
|Debian unstable | N         | Y             |
|OpenSuse        | N         | Y             |
|Arch            | N         | Y             |
|Fedora          | Y         | N             |

I stopped checking other systems since it became clear that only `goocanvas` will work on some and only `goocanvas-2.0` will work on others.

So, I propose this (I made a change to the commit in this PR):

```
if [[ $(pkg-config --exists goocanvas-2.0 2> /dev/null) ]] -o [[ $(pkg-config --exists goocanvas 2> /dev/null) ]]
```

See, I'm not really using `[[` -- autoconf will tun that into a `[`, which is POSIX-compliant.

Then, there is the problem of being able to detect `goocanvas` versions 3 and 4, but at least this partially fixes the problem.

I would guess that there must be a better way to do this...

**Comment by egallesio** [comment]
Hi @jpellegrini,

Thanks for this PR. I have applied it, but not regenerate the package (the procedure is not automatic). 

BTW, I'm porting since several days GTklos of GTK3. It is not yet finished. I didn't rally want to do this port,  because I thought it will take time (and it is really longer than I thought). Anyway, the port is almost finished for standard widgets. As soon as it is finished, I'll try to integrate goocanvas3. I hope it will not be too different from the old one. 
More on that soon.

**Comment by jpellegrini** [comment]
> BTW, I'm porting since several days GTklos of GTK3.

That's great! 
And in that case, I suppose the `if` in this PR would need to be updated, right?

**Comment by egallesio** [comment]
Yep. In fact, it seems that the name is now `libgoocanvas-3.0.so`, and its existence  can be tested by `pkg-config --exists goocanvas-3.0`
